### DISCUSSION TITLE
Removing ssid.toUpperCase();

### DISCUSSION THREAD

**Discussion by MattiaCC93** [khoih-prog/ESP_WiFiManager#98] [Q&A] [answered]
Can I remove the line of code "ssid.toUpperCase ();" without being afraid that everything will explode? 🫶

Line of code: https://github.com/khoih-prog/ESP_WiFiManager/blob/master/examples/ConfigOnDoubleReset/ConfigOnDoubleReset.ino#L814

**Comment by khoih-prog** [comment, marked as answer]
Totally OK. Just to change and display the AP SSID of `Config Portal` from `ESP_abcdef` to `ESP_ABCDEF` to read easier.
